Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: test
```

Comment:
That tells Make that it's not a real target (eg. there's not expected to be a local file called test, and running the rule will not create one). This means it will always run the command associated with the target. This probably explains it better than me: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html